docs/src/upgrade/release-3.6.x.asciidoc

```diff
@@ -776,3 +776,10 @@ semantics.
 
 See: link:https://tinkerpop.apache.org/docs/3.6.0/dev/provider/#_gremlin_semantics[Gremlin Semantics]
 See: link:https://tinkerpop.apache.org/docs/3.6.0/dev/developer/#_for_committers[Developer Documentation]
+
+===== Service Call API
```

Comment:
could you please add the link to the associated JIRA? Also could you please include a section for `element()`?